GUI/CalibrateTaskWindow.xaml.cs

```diff
@@ -22,25 +23,24 @@ public partial class CalibrateTaskWindow : Window
         private readonly ObservableCollection<ModTypeForTreeView> FixedModTypeForTreeViewObservableCollection = new ObservableCollection<ModTypeForTreeView>();
         private readonly ObservableCollection<ModTypeForTreeView> VariableModTypeForTreeViewObservableCollection = new ObservableCollection<ModTypeForTreeView>();
         private readonly ObservableCollection<ModTypeForLoc> LocalizeModTypeForTreeViewObservableCollection = new ObservableCollection<ModTypeForLoc>();
-        
-        public CalibrateTaskWindow()
-        {
-            InitializeComponent();
-            PopulateChoices();
-            
-            TheTask = new CalibrationTask();
-            UpdateFieldsFromTask(TheTask);
+        private readonly SearchModifications SearchMod = new SearchModifications();
 
-            this.saveButton.Content = "Add the Calibration Task";
+        public CalibrateTaskWindow() : this(null)
```

Comment:
@rmillikin i simplified the constructors for all the task windows cause I noticed code redundancy. let me know if u prefer having it the old way.